template/snippet.js

```diff
@@ -78,11 +78,10 @@
 
   // Load Analytics.js with your key, which will automatically
   // load the tools you've enabled for your account. Boosh!
-  {{{ load }}}
+  analytics.load("<%= settings.apiKey %>");
 
   // Make the first page call to load the integrations. If
   // you'd like to manually name or tag the page, edit or
   // move this call however you'd like.
-  {{{ page }}}
-
+  '<%= settings.page %>'
 })();
```

Comment:
this defines the latest snippet :) I'm going to bump the version to 4.0.0 in here once this is pushed